src/EnergyPlus/Vectors.cc

```diff
@@ -505,31 +493,27 @@ std::vector<int> PointsInPlane(Array1D<Vector> &BaseSurf, int const BaseSides, A
     return pointIndices;
 }
 
-Real64 CalcPolyhedronVolume(EnergyPlusData &state, Polyhedron const &Poly)
+Real64 CalcPolyhedronVolume(EnergyPlusData const &state, Polyhedron const &Poly)
```

Comment:
[src/EnergyPlus/Vectors.cc:508]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const